src/pipecat/services/openai/base_llm.py

```diff
@@ -460,8 +455,13 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
                 await self.push_frame(LLMFullResponseStartFrame())
                 await self.start_processing_metrics()
                 await self._process_context(context)
+            except asyncio.CancelledError:
+                raise
```

Comment:
If we are not doing anything, no need to handle it. By default it's already raised.